flepimop/gempyor_pkg/src/gempyor/config_validator.py

```diff
@@ -320,9 +339,15 @@ def verify_inference(cls, values):
         inference_present = values.get('inference') is not None
         start_date_groundtruth = values.get('start_date_groundtruth') is not None
         if inference_present and not start_date_groundtruth:
-            raise ValueError('Inference mode is enabled but no groundtruth dates are provided')
+            raise ValueError(
+                f'Inference mode is enabled, but no groundtruth dates are provided.'
+                f'Please provide groundtruth dates.'
+            )
         elif start_date_groundtruth and not inference_present:
-            raise ValueError('Groundtruth dates are provided but inference mode is not enabled')
+            raise ValueError(
+                f'Groundtruth dates are provided, but inference mode is not enabled.'
+                f'Please enable inference mode.'
+            )
```

Comment:
Kinda feels like this should be a single comparison and single error. Also, the error might be corrected by disabling, not necessarily enabling.